yarn.lock

```diff
@@ -16983,19 +16990,19 @@ uc.micro@^1.0.0, uc.micro@^1.0.1, uc.micro@^1.0.5:
   integrity sha512-8Y75pvTYkLJW2hWQHXxoqRgV7qb9B+9vFEtidML+7koHUFapnVJAZ6cKs+Qjz5Aw3aZWHMC6u0wJE3At+nSGwA==
 
 uglify-js@^3.1.4:
-  version "3.12.4"
-  resolved "https://registry.yarnpkg.com/uglify-js/-/uglify-js-3.12.4.tgz#93de48bb76bb3ec0fc36563f871ba46e2ee5c7ee"
-  integrity sha512-L5i5jg/SHkEqzN18gQMTWsZk3KelRsfD1wUVNqtq0kzqWQqcJjyL8yc1o8hJgRrWqrAl2mUFbhfznEIoi7zi2A==
+  version "3.12.2"
+  resolved "https://registry.yarnpkg.com/uglify-js/-/uglify-js-3.12.2.tgz#c7ae89da0ed1bb58999c7fce07190b347fdbdaba"
+  integrity sha512-rWYleAvfJPjduYCt+ELvzybNah/zIkRteGXIBO8X0lteRZPGladF61hFi8tU7qKTsF7u6DUQCtT9k00VlFOgkg==
 
 uid-number@0.0.6:
   version "0.0.6"
   resolved "https://registry.yarnpkg.com/uid-number/-/uid-number-0.0.6.tgz#0ea10e8035e8eb5b8e4449f06da1c730663baa81"
   integrity sha1-DqEOgDXo61uOREnwbaHHMGY7qoE=
 
 uikit@^3.5.9:
-  version "3.6.7"
-  resolved "https://registry.yarnpkg.com/uikit/-/uikit-3.6.7.tgz#db61ef4ab282d4d2a5faa44b9226f47a97ab9124"
-  integrity sha512-cbpaqmPKvMDHdPHHsdZFoX50rI5uqMuPkoPD6wVLQ1FiJ78MMPG0RKMtqB5lGU0fcI7yzJfDX+nqaSDu/bzVsQ==
+  version "3.6.5"
+  resolved "https://registry.yarnpkg.com/uikit/-/uikit-3.6.5.tgz#384df90f96a431d12d542b227da99ceff781c71e"
+  integrity sha512-ChsoZBuCC4y0CXqJ51tiIPS4zcKgj8yvXDl8njRyMBLCjNRFlHgSjem2T6pQsEEItoN+Vh7NE3iueD70A4LW4A==
```

Comment:
Drop all changes in the lockfile, the current lockfile in master is more up to date

Reply:
Done (`git checkout upstream/master -- yarn.lock`)